src/videojs-http-streaming.js

```diff
@@ -1062,11 +1062,34 @@ class VhsHandler extends Component {
 
     this.player_.tech_.on('keystatuschange', (e) => {
       if (e.status === 'output-restricted') {
-        this.masterPlaylistController_.blacklistCurrentPlaylist({
-          playlist: this.masterPlaylistController_.media(),
-          message: `DRM keystatus changed to ${e.status}. Playlist will fail to play. Check for HDCP content.`,
-          blacklistDuration: Infinity
-        });
+        const masterPlaylist = this.masterPlaylistController_.media();
+        const excludedHDPlaylists = [];
+
+        if (masterPlaylist && masterPlaylist.playlists) {
+          // Assume all HD streams are unplayable and exclude them from ABR selection, then perform
+          // a fastQualityChange to clear the buffer since it may already contain unplayable segments
+          masterPlaylist.playlists.forEach(playlist => {
+            if (playlist && playlist.attributes && playlist.attributes.RESOLUTION &&
+                playlist.attributes.RESOLUTION.height >= 720) {
+              if (!playlist.excludeUntil || playlist.excludeUntil < Infinity) {
+
+                playlist.excludeUntil = Infinity;
```

Comment:
I'm not sure if we need to check the current `excludeUntil` and can instead probably just set `playlist.excludeUntil = Infinity;`.

Reply:
I added this to make sure we only exclude/clear buffer/switch playlists once in the event we get multiple `output-restricted` events, which I have encountered in testing.